tests/console/clamav.pm

```diff
@@ -40,6 +40,12 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+
+    # Check Clamav version
+    # Jira ticket : SLE-16780 : upgrade Clamav SLE
+    my $current_ver = script_output("zypper se -s clamav");
```

Comment:
Please use `zypper_call('info clamav');` as per 8253f07c24, which would be shown like: https://openqa.suse.de/tests/5050066#step/firewalld/6

Reply:
Thanks @foursixnine for your suggestion, it is more useful and readable. I will change it. :+1: